packages/oui-radio/README.md

```diff
@@ -4,96 +4,79 @@
-  </oui-radio-description>
-</oui-radio>
-<oui-radio thumbnail label="Disabled [unchecked]" disabled>
-  <oui-radio-description>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque est ipsum, condimentum ornare condimentum quis, ultrices sit amet augue. Phasellus mollis dui quis nunc ultrices tempus. Praesent dignissim, felis in ornare euismod, augue elit mattis nibh, a tincidunt nunc enim et nulla. Nam ac blandit mauris. Donec semper tellus et felis viverra, in molestie lacus sodales. Proin fringilla vestibulum tempus. In feugiat risus mauris, tempus lacinia dolor posuere vel.
-  </oui-radio-description>
-</oui-radio>
-```
-
-## One-way binding properties
-
-### checked
-
-<oui-checkbox on-change="$ctrl.checked = $event.value" label="Click to check/uncheck the test radio"></oui-checkbox>
+### Description
 
 ```html:preview
-<oui-radio label="Test" checked="$ctrl.checked" disabled></oui-radio>
+<oui-radio text="Checked"
+    description="Checked radio"></oui-radio>
+<oui-radio text="Disabled"
+    description="Disabled radio"
+    disabled></oui-radio>
 ```
 
-### disabled
-
-<oui-checkbox on-change="$ctrl.disabled = $event.value" label="Click to disable the test radio"></oui-checkbox>
+### On change
 
 ```html:preview
-<oui-radio label="Test" checked disabled="$ctrl.disabled"></oui-radio>
+<div ng-init="$ctrl.value2 = 'a'">
+    <oui-radio text="Value A"
+        name="oui-radio-2"
+        model="$ctrl.value2"
+        value="'a'"
+        on-change="$ctrl.lastOnChangeValue = modelValue"></oui-radio>
+    <oui-radio text="Value B"
+        name="oui-radio-2"
+        model="$ctrl.value2"
+        value="'b'"
+        on-change="$ctrl.lastOnChangeValue = modelValue"></oui-radio>
+</div>
+
+<span>Last onChange value: {{ $ctrl.lastOnChangeValue }}</span>
 ```
 
-## Groups
-
-see `oui-radio-group` for details
+## API
+
+| Attribute     | Type                    | Binding | One-time Binding | Values                   | Default | Description
+| ----          | ----                    | ----    | ----             | ----                     | ----    | ----
+| text          | string                  | @       |                  |                          |         | checkbox text
+| description   | string                  | @?      |                  |                          |         | description text
+| id            | string                  | @?      | `true`           |                          |         | id attribute of the checkbox
+| name          | string                  | @?      | `true`           |                          |         | name attribute of the checkbox
+| disabled      | boolean                 | <?      |                  |                          | false   | disabled flag
+| model         | Object                  | =?      |                  |                          |         | current value of the checkbox and null is considered as `indeterminate`
```

Comment:
You're still talking about the *indeterminate* state, but radio didn't have one.